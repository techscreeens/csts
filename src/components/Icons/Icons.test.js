import React from "react";
import { render } from "react-testing-library";

import * as icons from "./";

describe("Icons", () => {
  it("should render icons correctly", () => {
    expect.assertions(icons.length);
    Object.values(icons).forEach(Icon =>
      expect(render(<Icon />).asFragment()).toMatchSnapshot()
    );
  });
});
