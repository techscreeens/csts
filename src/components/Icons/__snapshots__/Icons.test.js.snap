// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons should render icons correctly 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <svg
      fill="currentColor"
      height="20"
      viewBox="0 0 32 32"
      width="20"
    >
      <path
        d="M 15 4 C 14.476563 4 13.941406 4.183594 13.5625 4.5625 C 13.183594 4.941406 13 5.476563 13 6 L 13 7 L 7 7 L 7 9 L 8 9 L 8 25 C 8 26.644531 9.355469 28 11 28 L 23 28 C 24.644531 28 26 26.644531 26 25 L 26 9 L 27 9 L 27 7 L 21 7 L 21 6 C 21 5.476563 20.816406 4.941406 20.4375 4.5625 C 20.058594 4.183594 19.523438 4 19 4 Z M 15 6 L 19 6 L 19 7 L 15 7 Z M 10 9 L 24 9 L 24 25 C 24 25.554688 23.554688 26 23 26 L 11 26 C 10.445313 26 10 25.554688 10 25 Z M 12 12 L 12 23 L 14 23 L 14 12 Z M 16 12 L 16 23 L 18 23 L 18 12 Z M 20 12 L 20 23 L 22 23 L 22 12 Z "
        fill="currentColor"
      />
    </svg>
  </div>
</DocumentFragment>
`;

exports[`Icons should render icons correctly 2`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <svg
      fill="currentColor"
      height="20"
      viewBox="2 2 20 20"
      width="20"
    >
      <path
        d="M13 11V5.003A.999.999 0 0 0 12 4c-.552 0-1 .438-1 1.003V11H5.003A.999.999 0 0 0 4 12c0 .552.438 1 1.003 1H11v5.997A.999.999 0 0 0 12 20c.552 0 1-.438 1-1.003V13h5.997A.999.999 0 0 0 20 12c0-.552-.438-1-1.003-1H13z"
        fill="currentColor"
      />
    </svg>
  </div>
</DocumentFragment>
`;
